backend/fleet_management/tests/test_drives.py

```diff
@@ -0,0 +1,96 @@
+from datetime import date
+
+from django.contrib.auth.models import Group
+from django.test import override_settings
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from fleet_management.constants import Groups
+from fleet_management.factories import (
+    DriveFactory,
+    PassengerFactory,
+    CarFactory,
+    ProjectFactory,
+    UserFactory,
+)
+from fleet_management.models import Drive
+
+
+class DrivesApiTest(APITestCase):
+    def setUp(self):
+        self.url = reverse("drives")
+        self.passengers = PassengerFactory.create_batch(size=10)
+        self.car = CarFactory()
+        self.project = ProjectFactory(country="UA")
+        self.driver = UserFactory()
+        self.driver.groups.set(Group.objects.filter(name=Groups.Driver.name))
+
+        self.drives = DriveFactory.create_batch(
+            size=10,
+            car=self.car,
+            driver=self.driver,
+            date=date.today().isoformat(),
+            project=self.project,
```

Comment:
Do we rely on `self.project` in any way? (and `self.car`, and `date.today().isoformat()`)

I guess not, so we could remove `self.car` and `self.project` altogether and build that as
```
self.drives = DriveFactory.create_batch(size=10, driver=self.driver)
```

Reply:
I have checked, it's working without it. Fixed.